source/context_menus/_context_menu_stats.js

```diff
@@ -0,0 +1,58 @@
+const { PermissionFlagBits, InteractionContextType } = require('discord.js');
+const { UserContextMenuWrapper } = require('../classes');
+
+const mainId = "User Stats";
+module.exports = new UserContextMenuWrapper(mainId, null, false, [ InteractionContextType.Guild ], 3000,
+	/** Specs */
+	(interaction, database, runMode) => {
+		const target = interaction.targetMember;
+		if (target.id == interaction.client.user.id) {
+			// BountyBot
+			buildCompanyStatsEmbed(interaction.guild, database).then(embed => {
+				interaction.reply({
+					embeds: [embed],
+					ephemeral: true
+				});
+			})
+		} else {
+			// Other Hunter
```

Comment:
Please create a function for this and the own stats embeds so that functionality can stay in step with the slash command.

Reply:
This will be moved to its own tasking